stats.cpp

```diff
@@ -1,5 +1,20 @@
 #include "stats.h"
-
-Stats Statistics::ComputeStatistics(const std::vector<___>& ) {
-    //Implement statistics here
+#include <bits/stdc++.h>
+namespace Statistics {
+    
+    Stats ComputeStatistics(const std::vector<float>& floatVector) {
+        Stats statsObj;
+        auto size_floatVector = floatVector.size();
+        if(size_floatVector == 0)
+        {
+            statsObj.average = std::numeric_limits<float>::quiet_NaN();
+            statsObj.max = std::numeric_limits<float>::quiet_NaN();
+            statsObj.min = std::numeric_limits<float>::quiet_NaN();
```

Comment:
Another way to do this is to write a constructor in the Stats structure to initialize everything to NaN. That way, we don't need to worry about un-initialized variables in our program.

Reply:
Yes, I have completed this.